src/test/java/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImplTest.java

```diff
@@ -113,101 +113,6 @@ public void displayName() throws IOException {
         assertEquals(EXPECTED_DISPLAY_NAME, CredentialsNameProvider.name(new CertificateCredentialsImpl(null, "abc123", null, "password", storeSource)));
     }
 
-    @Test
```

Comment:
the code these tests where testing has been deleted.
Whilst we could leave a test that shows the behaviour doesn't work, I don't see a point in that)